sass/components/_toggle.scss

```diff
@@ -48,10 +49,10 @@ $toggle-default-font-size: 0.75rem;
 
     > .handle {
         position: absolute;
-        top: 0.4rem;
-        left: 0.4rem;
-        width: 2rem;
-        height: 2rem;
+        top: 0.2rem;
+        left: 0.25rem;
+        width: 1.15rem;
+        height: 1.15rem;
```

Comment:
This is where the css of the larger toggle breaks. It may be better to get rid of this mixin and instead have the caller handle the different sizes explicitly.

Reply:
since you are able now to pass a custom class to the toggle component, maybe it is a better idea to stylish it within a css file in the scope where the Toggle component is created. In this case it could be create a style in the marketplace context.